### ISSUE TITLE
req.originalUrl is modified

### ISSUE THREAD

**Issue by runekm** [OptimalBits/redbird#141]
If I register this route:
proxy.register("mydomain.com", "http://localhost:3000");
and navigate to f.ex.:
http://mydomain.com///mypage////
in my browser, then
http://localhost:3000/mypage/
is served from Redbird.

In my app req.originalUrl will be /mypage/, so I have no way to know that the user is at  http://mydomain.com///mypage////, and not http://mydomain.com/mypage/

It would be better if Redbird just had served http://localhost:3000///mypage//// instead, as I then could've shown a 404 page or used a 301 redirect.

**Comment by manast** [comment]
you are assuming that redbird knows that the original url is ```//mypage///```, have you checked that the browser is not striping all the extra / before sending the request to the server?

**Comment by runekm** [comment]
Yes, the browser is displaying http://mydomain.com///mypage//// in the address bar while req.originalUrl in my node app says /mypage/

If I navigate directly to http://localhost:3000///mypage//// instead (not through Redbird), req.originalUrl says ///mypage////

**Comment by runekm** [comment]
After a little bit of digging around, it seams that this behaviour is caused by
.replace(/\/+/g, '/') on line 191 of
/node_modules/http-proxy/lib/http-proxy/common.js

**Comment by manast** [comment]
ok, then it is an issue for the http-proxy project: https://github.com/nodejitsu/node-http-proxy
